src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -890,367 +667,410 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
+                            id={phaseErrorId}
+                            role="alert"
+                            aria-live="polite"
+                            $flexDirection={"row"}
+                            $mb={20}
+                          >
                             <Icon
-                              $color="red"
+                              $color={"red"}
                               name="content-guidance"
                               verticalAlign="bottom"
                             />
-                            <OakP $color="red">
+                            <OakP $color={"red"}>
                               Select a school phase to view the curriculum
                             </OakP>
                           </Flex>
                         )}
-
-                        {showKS4OptionError && (
+                        {showKS4OptionError ? (
                           <Flex
                             id={ks4OptionErrorId}
-                            $flexDirection="row"
+                            role="alert"
+                            aria-live="polite"
+                            $flexDirection={"row"}
                             $mb={20}
                           >
                             <Icon
-                              $color="red"
+                              $color={"red"}
                               name="content-guidance"
                               verticalAlign="bottom"
                             />
-                            <OakP $color="red">
+                            <OakP $color={"red"}>
                               Select a KS4 option to view the curriculum
                             </OakP>
                           </Flex>
+                        ) : (
+                          ""
                         )}
-
+                        <OakHeading
+                          id={schoolPhaseInputId}
+                          tag={"h4"}
+                          $font={"heading-6"}
+                          $mb="space-between-s"
+                          data-testid="phase-picker-heading"
+                        >
+                          Choose a school phase
+                        </OakHeading>
                         <OakFlex
-                          role="radiogroup"
+                          radioGroup="radiogroup"
                           aria-labelledby={schoolPhaseInputId}
                           aria-required="true"
                           aria-describedby={
                             showPhaseError ? phaseErrorId : undefined
                           }
-                          $flexDirection="column"
-                          $gap="space-between-s"
+                          $flexDirection={"column"}
+                          $gap={"space-between-s"}
                         >
+                          <CurriculumModalCloseButton
+                            ariaLabel="Close phase picker modal"
```

Comment:
Again this is pre-existing code but I would advise getting rid of the word 'modal' here - it's a bit redundant and some users will have no idea what a modal is.